goose_test.go

```diff
@@ -8,11 +8,11 @@ import (
 
 func TestDefaultBinary(t *testing.T) {
 	commands := []string{
-		"go build -i -o goose ./cmd/goose",
-		"./goose -dir=examples/sql-migrations sqlite3 sql.db up",
-		"./goose -dir=examples/sql-migrations sqlite3 sql.db version",
-		"./goose -dir=examples/sql-migrations sqlite3 sql.db down",
-		"./goose -dir=examples/sql-migrations sqlite3 sql.db status",
+		"go build -i -o ./bin/goose ./cmd/goose",
```

Comment:
maybe we should have `testbin` that is created in the test.. on master running `go test` makes the working tree dirty, which is not great.